recipes/ipypdf/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "0.0.1" %}
+
+package:
+  name: ipypdf
+  version: {{ version }}
+
+source:
+  url: https://github.com/joelstansbury/ipypdf/archive/0206fb7.tar.gz
+  sha256: ec1e9cfd2db9e45963e2664d86acfdaf7b5c4568e654ceb3c6f60d7a29f65f3d
+
+build:
+  script: {{ PYTHON }} -m pip install -vv . --no-deps
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+    - pip
+    - ipycanvas
+    - ipycytoscape
+    - ipyevents
+    - ipytree
+    - ipywidgets
+    - jupyterlab >=3.0
+    - numpy <=1.19.3,>=1.13  # Constrained by paddlepaddle
+    - pandas
+    - pdf2image
+    - poppler
+    - pytesseract
+    - spacy
+    - spacy-model-en_core_web_lg
+    - tesseract
+    - traitlets
+
+test:
+  source_files:
+    - tests
+  requires:
+    - pytest
```

Comment:
i _do_ recommend adding `- pip` to the `requires` and running `pip check` before your testing your code: with a stack this deep, making sure all of those stays accurate and useful is part of the job.